recipes/boost/all/conanfile.py

```diff
@@ -760,7 +761,7 @@ def _clean(self):
         ]
         for d in clean_dirs:
             if os.path.isdir(d):
-                self.output.warn("removing '%s'".format(d))
+                self.output.warn("removing '{}'".format(d))
```

Comment:
maybe it would be better to replace it with f-string?
https://pylint.pycqa.org/en/latest/user_guide/messages/convention/consider-using-f-string.html

Reply:
Ok. I´ll wait for https://github.com/conan-io/conan-center-index/pull/12131 to be merged, then merge those changes here together with your suggestion.

Reply:
https://github.com/conan-io/conan-center-index/runs/7608969259?check_suite_focus=true
Linter suggests replacing lots of "format" usages in boost

Reply:
Yes, it is probably more python-friendly, but those are not errors and are not enforced by the linter right now (they are not required for the PR to be merged).

My main consideration here is that a PR to `boost` takes around ~15hours to build all the versions and configurations, so I prefer to apply several changes together than triggering a new build for minor details/fixes. This is valid for any PR here, but especially for those that take so much time.

So I prefer to wait for #12131, and then apply it together with all the changes to f-strings.